src/applications/vaos/utils/selectors.js

```diff
@@ -334,3 +334,6 @@ export const selectSystemIds = state =>
 
 export const selectIsCernerOnlyPatient = state =>
   !!selectFacilities(state)?.every(f => f.isCerner);
+
+export const selectAtleastOneCernerFacility = state =>
```

Comment:
I wonder if we should maybe call this `selectIsCernerPatient` to go along with the existing `selectIsCernerOnlyPatient`? Gonna tag @jbalboni and @narin since this file is theirs.

Reply:
That sounds ok to me.

Also, I'd rather we move these selectors into platform code if you're using them from other applications. That way we don't accidentally break your code with the platform team reviewing it.

Reply:
@erikphansen @jbalboni I have moved the selectors, their unit tests and changed their imports over to `platform/user/selectors`